frontend/packages/client/src/components/ProposalInformation.js

```diff
@@ -6,20 +6,72 @@ import { useVotingResults, useWindowDimensions } from '../hooks';
 import useMediaQuery, { mediaMatchers } from '../hooks/useMediaQuery';
 import Tooltip from './Tooltip';
 
-const InfoBlock = ({ title, content }) => {
+function truncate(str) {
+  return str.substr(0, 3) + '...' + str.substr(10, str.length);
+}
+
+const BlockieWithAddress = React.forwardRef(
+  ({ creatorAddr, isCoreCreator }, ref) => {
+    const [addr, setAdd] = useState(creatorAddr);
+
+    const { width } = useWindowDimensions();
+
+    useEffect(() => {
+      if (ref?.current.clientWidth <= 223 && creatorAddr === addr) {
```

Comment:
this number might vary depending on the contents of the address, for example if there is a lot of W (the widest letter) vs a lot of I and 1 etc - probably works well enough for most addresses though or maybe add a bit of extra padding (250?) in case of longer addresses. Otherwise youll want to give the address `white-space: nowrap` and then calculate if the width of the address is greater than the width of the parent container and then truncate